internals/secrethub/write_test.go

```diff
@@ -192,6 +194,37 @@ func TestWriteCommand_Run(t *testing.T) {
 			},
 			err: errClipAndInFile,
 		},
+		"multiline and clip or in file": {
```

Comment:
It is not clear to me what `or in file` means in the name of this test.

If it is a test that sets both the multiline and clip flags, `multiline and clip` should suffice as a test name.